UnitTests/Core/BorderTests.cs

```diff
@@ -557,57 +557,5 @@ public void BorderOnControlWithNoChildren ()
 
 			Assert.Null (Record.Exception (() => label.Redraw (label.Bounds)));
 		}
-
-		[Fact, AutoInitShutdown]
-		public void BorderStyle_And_DrawMarginFrame_Gets_Sets ()
```

Comment:
I hope you also maintain this unit test. I think you have some errors because of the `View.Clear` method.